core/src/app/command/runnable.rs

```diff
@@ -78,6 +78,12 @@ impl From<(String, String)> for AutocompleteSuggestion {
     }
 }
 
+impl From<(&'static str, &'static str)> for AutocompleteSuggestion {
```

Comment:
There's a few other string flavours of this. Adding one for static strings makes building these in unit tests a lot easier.